src/parser/TranspileState.ts

```diff
@@ -1,87 +1,66 @@
 import { SourceNode } from 'source-map';
 import type { Range } from 'vscode-languageserver';
-import type { BrsFile } from '../files/BrsFile';
-import type { Token } from '../lexer/Token';
-import type { ClassStatement } from './Statement';
+import type { BsConfig } from '../BsConfig';
 
 /**
  * Holds the state of a transpile operation as it works its way through the transpile process
  */
 export class TranspileState {
     constructor(
         /**
-         * The BrsFile that is currently being transpiled
+         * The absolute path to the source location of this file. If sourceRoot is specified,
+         * this path will be full path to the file in sourceRoot instead of rootDir.
+         * If the file resides outside of rootDir, then no changes will be made to this path.
          */
-        public file: BrsFile
+        public srcPath: string,
```

Comment:
Use name `srcPath` to align with the path variable renames in the upcoming PR #329